modules/programs/fzf.nix

```diff
@@ -198,7 +198,7 @@ in {
     # Still needs to be initialized after oh-my-zsh (order 800), otherwise
     # omz will take precedence.
     programs.zsh.initContent =
-      mkIf cfg.enableZshIntegration (mkOrder 810 zshIntegration);
+      mkIf cfg.enableZshIntegration (mkOrder 910 zshIntegration);
```

Comment:
Could maybe use a comment similar to the one about `oh-my-zsh`?

Reply:
yeah i sorta forgot to update the comment :|